### ISSUE TITLE
wpcolt.com [Breakage] [FETL] [reopen]

### ISSUE THREAD

**Issue by kulfoon** [ryanbr/fanboy-adblock#688]
Reopening #652, because although it has been fixed on Opera, but it is still here on Firefox.

**Comment by ryanbr** [comment]
I don't use opera, and my last comment still stands, so unless the favicon is hosted outside of wpcolt.com, there isn't much I'm seeing. Creating a new ticket won't help.

**Comment by kulfoon** [comment]
From https://github.com/ryanbr/fanboy-adblock/issues/652#issuecomment-437930301

> THEtomaso :
> 
> `/favicon.ico$image,third-party,domain=~wpcolt.com` is needed for 'Fanboy's Enhanced Tracking List', since this blocking rule is partially responsible for causing the overlay.

From https://github.com/ryanbr/fanboy-adblock/issues/652#issuecomment-439582881

> ryanbr :
> 
> `https://cdn.wpcolt.com/wp-content/uploads/2018/05/favicon.ico`
> 
> I don't see it being blocked, its a first-party element.

As seen THEtomaso clearly have written `third-party`, and not first-party, and the third-party is the other one, this one:
`https://apps.facebook.com/favicon.ico`

blocked by 

`/favicon.ico$third-party,image,domain=~stackexchange.com|~stackoverflow.com|~askubuntu.com|~reddit.com|~4chan.org|~twitter.com|~live.com|~espn.com`

from Fanboy’s Enhanced Tracking List:

![ss061](https://user-images.githubusercontent.com/43844449/48977142-e1dd4180-f094-11e8-83b0-dbfc910a6630.png)

![ss062](https://user-images.githubusercontent.com/43844449/48977128-c3774600-f094-11e8-9e23-8eb95e1d6ada.png)

**Comment by ryanbr** [comment]
If facebook want to track with a favicon, it'll be blocked. I'm leaving it.
